subgraphs/nft-market/mappings/index.ts

```diff
@@ -102,6 +103,10 @@ export function handleAskNew(event: AskNew): void {
   if (token == null) {
     token = new NFT(tokenConcatId);
     token.tokenId = event.params.tokenId;
+    // If collection is Pancake Bunnies --> try fetching the bunnyId
+    if (event.params.collection.toHexString() == PANCAKE_BUNNIES_ADDRESS) {
```

Comment:
```suggestion
    if (event.params.collection.equals(Address.fromString(PANCAKE_BUNNIES_ADDRESS))) {
```
Not 100% of the syntax but should be something along these lines